src/super_gradients/recipes/training_hyperparams/coco2017_yolo_nas_train_params.yaml

```diff
@@ -8,7 +8,10 @@ warmup_initial_lr:  1e-6
 lr_warmup_steps: 1000
 lr_warmup_epochs: 0
 
-initial_lr:  2e-4
+initial_lr:
```

Comment:
If we have a single LR for all model, maybe let's not introduce mapping?

Reply:
fixed 966180d62ef7e8c1a5eb1e2a98a2f9a8783c9af3